src/EditorFeatures/Core/AutomaticInsertionOfAbstractOrInterfaceMembers/AutomaticInsertionOfAbstractOrInterfaceMembersOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.AutomaticInsertionOfAbstractOrInterfaceMembers
+{
+    internal static class AutomaticInsertionOfAbstractOrInterfaceMembersOptions
+    {
+        // This value is only used by Visual Basic, and so is using the old serialization name that was used by VB.
+        public static readonly PerLanguageOption2<bool> AutomaticInsertionOfAbstractOrInterfaceMembers = new(
```

Comment:
For features where we have a single bool that's the feature enabling, should we just call the .NET side of property "Enabled"? As it is the fully qualified name of this member is Microsoft.CodeAnalysis.AutomaticInsertionOfAbstractOrInterfaceMembers.AutomaticInsertionOfAbstractOrInterfaceMembersOptions.AutomaticInsertionOfAbstractOrInterfaceMembers; feels a bit redundant!